src/StateBridge.sol

```diff
@@ -62,6 +67,14 @@ contract StateBridge is FxBaseRootTunnel, Ownable {
     /// @notice Thrown when the caller of `sendRootMultichain` is not the WorldID Identity Manager contract.
     error NotWorldIDIdentityManager();
 
+    ///////////////////////////////////////////////////////////////////
+    ///                          MODIFIERS                          ///
+    ///////////////////////////////////////////////////////////////////
+    modifier onlyWorldIDIdentityManager() {
+        if (msg.sender != worldIDAddress) revert NotWorldIDIdentityManager();
```

Comment:
Please use a braced `if` style.

Reply:
that's what solidity devs on Twitter would call `the boomer way`, but I get why you say that. I'm just personally used to this format since it's in all codebases and libraries I've seen pretty much. But consistency is important.

Reply:
using braced style for this codebase now